src/Data/Traversable.js

```diff
@@ -0,0 +1,48 @@
+/* global exports */
+"use strict";
+
+// module Data.Traversable
+
+function Cont (fn) {
```

Comment:
Would it be appropriate to move these inside the definition of `traverseArrayImpl`?